### PR TITLE
Update int datatype tests

### PR DESCRIPTION
**Pull request by DonAurelio**
* We updated the int data type test.
* It validates correct geneartion of the type in filter and sink.
* It validates consistency and correctness of values through the state machine.

### CODE REVIEW

File: test/data_types/cases_int/1.source_callbacks.c [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #include <stdint.h>
 
 void btx_push_usr_messages(void *btx_handle, void *usr_data, btx_source_status_t *status) {
-    btx_push_message_event(btx_handle, UINT64_MAX, UINT64_MAX, UINT32_MAX, INT64_MAX, INT64_MAX, INT32_MAX);
+    btx_push_message_event(btx_handle, 0, 0, 0, INT64_MIN, INT64_MIN, INT32_MIN);
```
Comment (TApplencourt) [inline test/data_types/cases_int/1.source_callbacks.c:5]: why 0, 0, 0?

Comment (DonAurelio) [reply test/data_types/cases_int/1.source_callbacks.c:5]: `stdint.h` does not define `UINT64_MIN` neither UINT32_MIN.

**Comment by DonAurelio** [comment]
I think we are good to go

File: test/data_types/cases_int/1.source_callbacks.c (hunk 1 above)
Comment (TApplencourt) [reply test/data_types/cases_int/1.source_callbacks.c:5]: Oh funny!

**Review by TApplencourt — APPROVED** [review]

**Review by TApplencourt — APPROVED** [review]

**Pull request merged**